src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,114 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    public static final int VALID_AGE = 18;
+    public static final int LOWER_18 = 17;
+    public static final String VALID_LOGIN = "login";
+    public static final String VALID_PASSWORD = "password";
+    public static final String WRONG_PASSWORD = "wrong";
+    private static RegistrationServiceImpl service;
```

Comment:
use abstraction, and clear name
```suggestion
    private static RegistrationService registrationService;
```